### ISSUE TITLE
Why does pad_last_frame() use last frame?

### ISSUE THREAD

**Issue by karan-dalal** [THUDM/CogVideo#643]
### System Info / 系統信息

N/A

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

N/A

### Expected behavior / 期待表现

Hi, thank you for releasing this codebase and weights! https://github.com/THUDM/CogVideo/blob/aa240dc675fd7e73a16b65e5b5e43e6a7638b0bd/sat/data_video.py#L150

I had some questions about this choice.
1. Was this tested against making the padded screen black?
2. When finetuning, what is the maximum padding that we should add? I assume we shouldn't train with a 3 second video and a 6 second video, for example, but is a 5 and 6 second video okay?

**Comment by zRzRzRzRzRzRzR** [comment]
During model pretraining, for videos that do not meet the required duration, we pad the last frame of the video instead of using pad0. This is an engineering measure during training.

**Comment by karan-dalal** [comment]
See my two questions:
1. Was this tested against making the padded screen black?
2. When finetuning, what is the maximum padding that we should add? I assume we shouldn't train with a 3 second video and a 6 second video, for example, but is a 5 and 6 second video okay?

**Comment by zRzRzRzRzRzRzR** [comment]
The maximum padding should be filled to the longest video in your set script.
Suppose your maximum training length is 6 seconds.
All your videos (using our script) will be padded to 6 seconds. Pad the last frame, not black.
We have not tested setting the padded screen to black in our group.

**Comment by zRzRzRzRzRzRzR** [comment]
Assuming your original data has 3 seconds, 5 seconds, and 6 seconds, the data for 3 seconds and 5 seconds will both pad the last frame to 6 seconds during the LoRa fine-tuning process.

**Comment by karan-dalal** [comment]
Would you expect bad quality if we use such variable lengths during finetuning process?

**Comment by zRzRzRzRzRzRzR** [comment]
It can be said that the effect is definitely not as good as fine-tuning with 6 seconds for all. However, I believe this is because the model was trained with only one duration of 6 seconds in the postrain phase (5 seconds / 10 seconds for cogvideox1.5). The other data is too scarce.
